content/crutchcorn/posts/how-computers-speak/index.md

```diff
@@ -231,7 +231,7 @@ int main() {
 }
 ```
 
-This code simply says, "print the number 185 to the screen so the user can see it". **To do the same in assembly requires a massive amount of knowledge about the system** you're intending to run code on, due to the lack of portability granted by higher-level languages.
+This code simply says, "print the number 185 to the screen so the user can see it". **To do the same in assembly requires a massive amount of knowledge about the system** you're intending to run code on, due to the lack of portability granted by lower-level languages.
```

Comment:
Let's revert this change or rephrase this sentence.

The intended message was "higher level languages has portability that assembly does not, leading to more knowledge required to write assembly"

Reply:
Hi! Right now, the sentence says that higher-level languages have a "lack" of portability, implying that higher-level languages *aren't* as portable as lower-level languages, which is the opposite of the intended point. 
I'm happy to revert the change if you don't agree.

Reply:
For sure. I see both sides, but that ambiguity is probably a sign we should rewrite this sentence. 

Any chance you wanna tackle that real quick then we can merge? ☺️

Reply:
No problem!